integration_test/config/test_packages.yml

```diff
@@ -226,3 +227,63 @@ test_cases:
   inputs: [ image.tiff ]
 - arguments: [ imsave ]
   outputs: [ image2.tiff ]
+---
+script: run_netcdf4.py
+libraries: [ netCDF4 ]
+test_cases:
+- arguments: [ dataset_read ]
+  inputs: [ testrh.nc ]
+- arguments: [ dataset_write ]
+  outputs: [ data.nc ]
+- arguments: [ dataset_append ]
+  inputs: [ testrh.nc ]
+  outputs: [ testrh.nc ]
+---
+script: run_xarray.py
+libraries: [ xarray ]
+test_cases:
+- arguments: [ open_dataset ]
+  inputs: [ soilPropertiesRhineMeuse30min.nc ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ open_mfdataset_glob ]
+# recipy logs the input files in addition to the glob pattern
+  inputs: [ '*PropertiesRhineMeuse30min.nc', soilPropertiesRhineMeuse30min.nc, topoPropertiesRhineMeuse30min.nc ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ open_mfdataset_list ]
+  inputs: [ soilPropertiesRhineMeuse30min.nc, topoPropertiesRhineMeuse30min.nc ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ open_rasterio ]
+  inputs: [ image.tiff ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ open_dataarray ]
+  inputs: [ data_array.nc ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ dataset_to_netcdf ]
+  outputs: [ data.nc ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ dataarray_to_netcdf ]
+  outputs: [ data.nc ]
+  skip_py_version: [ 3.4 ]
+- arguments: [ save_mfdataset ]
+  outputs: [ data1.nc, data2.nc ]
+  skip_py_version: [ 3.4 ]
+---
+script: run_iris.py
+# the test package can't detect the iris installation
+#libraries: [ iris ]
```

Comment:
No, I don't think I really looked into this. The problem is that iris is not in the `pkg_resources.working_set`. That is as far as I got, because I had no idea where to start for finding out why this happens. I can confirm that recipy correctly logs the version of iris.

Reply:
I'll look into this

Reply:
I've done a fairly deep investigation of this in #194, but I'm tired now and going to bed. Let me know what you think.